<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuca: Nucleotide Compression Algorithms</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>191</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCompression">
      <attribute name="title">
       <string>Compression</string>
      </attribute>
      <widget class="QLabel" name="labelInputCompression">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File to Compress:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelOutputCompression">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File Compressed Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="compressFileIn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>241</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="compressFileOut">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>271</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExploreCom">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="compressButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Compress</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabDecompression">
      <attribute name="title">
       <string>Decompression</string>
      </attribute>
      <widget class="QLineEdit" name="decompressionFileIn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>241</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="decompressionFileOut">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>271</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelInputDecompressing">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>128</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File to Decompress:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelOutputDecompressing">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>80</y>
         <width>168</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>File Decompressed Name:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExploreDec">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="decompressButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Decompress</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
